payments/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db  import models
+
+from users.models       import Users, Addresses
+from orders.models      import Orders
+
+
+class Payments(models.Model):
+    first_name          =   models.CharField(max_length = 200)
+    last_name           =   models.CharField(max_length = 200)
+    address             =   models.CharField(max_length = 500)
+    appartment_type     =   models.CharField(max_length = 500)
+    city                =   models.CharField(max_length = 200)
+    county_region       =   models.CharField(max_length = 100)
+    postcode            =   models.CharField(max_length = 50)
+    phone_number        =   models.CharField(max_length = 200)
+    address          =   models.ForeignKey("users.Addresses",on_delete = models.CASCADE)
+
+    class Meta :
+        db_table = "payments"
+
+class Shipping(models.Model):
+    delivery_method     =   models.CharField(max_length = 200)
```

Comment:
이것도 몇가지 정해진 옵션들 중 고르게 될텐데 정규화 해주시면 좋을 것 같네요.

Reply:
```suggestion
    delivery_method_id     =   models.ForeignKey(max_length = 200)
```

class DeliveryMethod

Reply:
이 사이트가 요일마다 다른지는 모르겠지만 현재 지금 확인한결과 배송방법은 유일한 한가지 밖에 없는걸로 확인이 되었습니다.
옵션이 없으므로  테이블 그대로 사용하려고 합니다.